Octokit/Models/Response/IssueComment.cs

```diff
@@ -63,19 +63,4 @@ internal string DebuggerDisplay
             get { return string.Format(CultureInfo.InvariantCulture, "Id: {0} CreatedAt: {1}", Id, CreatedAt); }
         }
     }
-
-    public enum IssueCommentSort
```

Comment:
I think this is a bug...   Looks like the `IssueCommentRequest` object uses `PullRequestReviewCommentSort` but should actually use `IssueCommentSort`